packages/run-protocol/test/psm/test-psm.js

```diff
@@ -0,0 +1,190 @@
+  const [psmBundle] = await Promise.all([makeBundle(psmRoot)]);
+  t.context.bundles = { psmBundle };
+  const { makeFar, makeNear: makeRemote } = makeLoopback('zoeTest');
+  const { zoeService, feeMintAccess: nonFarFeeMintAccess } = makeZoeKit(
+    makeFakeVatAdmin(setJig, makeRemote).admin,
+  );
+  /** @type {ERef<ZoeService>} */
+  const zoe = makeFar(zoeService);
+  t.context.zoe = zoe;
+  trace('makeZoe');
+  const feeMintAccess = await makeFar(nonFarFeeMintAccess);
+  t.context.feeMintAccess = feeMintAccess;
+
+  const runIssuer = await E(zoe).getFeeIssuer();
+  const runBrand = await E(runIssuer).getBrand();
+  t.context.runKit = { runIssuer, runBrand };
+  const anchorKit = makeIssuerKit('aUSD');
+  t.context.anchorKit = anchorKit;
+
+  const { brand: anchorBrand } = anchorKit;
+  const psmInstall = await E(zoe).install(psmBundle);
+  t.context.installs = { psmInstall };
+  const mintLimit = AmountMath.make(anchorBrand, MINT_LIMIT);
+  t.context.mintLimit = mintLimit;
+  t.context.terms = {
+    anchorBrand,
+    main: { WantStableFeeBP, GiveStableFeeBP, MintLimit: mintLimit },
+  };
+});
+
+test('simple trades', async t => {
+  const {
+    zoe,
+    feeMintAccess,
+    terms,
+    installs: { psmInstall },
+    runKit: { runIssuer, runBrand },
+    anchorKit: { brand: anchorBrand, issuer: anchorIssuer, mint: anchorMint },
+  } = t.context;
+  const { publicFacet } = await E(zoe).startInstance(
+    psmInstall,
+    harden({ AUSD: anchorIssuer }),
+    terms,
+    harden({ feeMintAccess }),
+  );
+  const give = AmountMath.make(anchorBrand, 200n * 1_000_000n);
+  const seat1 = await E(zoe).offer(
+    E(publicFacet).makeSwapInvitation(),
+    harden({ give: { In: give } }),
+    harden({ In: anchorMint.mintPayment(give) }),
+  );
+  const runPayout = await E(seat1).getPayout('Out');
+  const expectedRun = minusFee(give, runBrand);
+  const actualRun = await E(runIssuer).getAmountOf(runPayout);
+  t.deepEqual(actualRun, expectedRun);
+  const liq = await E(publicFacet).getCurrentLiquidity();
+  t.true(AmountMath.isEqual(liq, give));
+  trace('get stable', { give, expectedRun, actualRun, liq });
+  const runGive = AmountMath.make(runBrand, 100n * 1_000_000n);
+  const [runPayment, _moreRun] = await E(runIssuer).split(runPayout, runGive);
+  const seat2 = await E(zoe).offer(
+    E(publicFacet).makeSwapInvitation(),
+    harden({ give: { In: runGive } }),
+    harden({ In: runPayment }),
+  );
+  const anchorPayout = await E(seat2).getPayout('Out');
+  const actualAnchor = await E(anchorIssuer).getAmountOf(anchorPayout);
+  const expectedAnchor = AmountMath.make(
+    anchorBrand,
+    minusFee(runGive, runBrand).value,
+  );
+  t.deepEqual(actualAnchor, expectedAnchor);
+  const liq2 = await E(publicFacet).getCurrentLiquidity();
+  t.deepEqual(AmountMath.subtract(give, expectedAnchor), liq2);
+  trace('get anchor', { runGive, expectedRun, actualAnchor, liq2 });
+});
+
+test('limit', async t => {
+  const {
```

Comment:
This tests trying to extract more than has been added. I also want to see trying to get more than the `mintLimit`

Reply:
It's contributing more than `mintLimit` in `anchor` (line 168), so that is what it's testing.